bin/runtime/src/lib.rs

```diff
@@ -701,9 +699,17 @@ parameter_types! {
     pub Schedule: pallet_contracts::Schedule<Runtime> = Default::default();
 }
 
+pub struct ZeroRandomness;
+
+impl Randomness<Hash, AlephBlockNumber> for ZeroRandomness {
+    fn random(_: &[u8]) -> (Hash, AlephBlockNumber) {
+        Default::default()
```

Comment:
just to find out myself in the context: the whole change is because pallet contracts dropped support for randomness as a stable feature for contracts?

Reply:
Not really, the change is because we do not this pallet and in the contracts the support for randomness can be replaced by a dummy implementation since its dropped